plugin/src/main/java/com/denizenscript/denizen/tags/core/TextTagBase.java

```diff
@@ -36,8 +36,9 @@ public TextTagBase() {
         // @description
         // Returns a special chat code that makes the following text display the input hover text when the mouse is left over it.
         // This tag must be followed by an <&end_hover> tag.
-        // For example: - narrate "There is a <&hover[you found it!]>secret<&end_hover> in this message!"
         // Note that this is a magic Denizen tool - refer to <@link language Denizen Text Formatting>.
+        // @example
+        // - narrate "There is a <&hover[you found it!]>secret<&end_hover> in this message!"

```

Comment:
examples should have `# comments` on them noting what the usage does in practice